src/components/AssetInfo/TokenSideBar.test.tsx

```diff
@@ -4,7 +4,15 @@ import TokenSideBar from "./TokenSideBar";
 
 describe("tokenSideBar", () => {
   it("should have a Manage Asset heading", () => {
-    const wrapper = mount(<TokenSideBar isSideBarExpand={true} handler={() => {}} />);
+    const wrapper = mount(
+      <TokenSideBar
+        isSideBarExpand={true}
+        holderAddress="0xA"
+        beneficiaryAddress="0xB"
+        approvedBeneficiaryAddress=""
+        handler={() => {}}
+      />
+    );
```

Comment:
write more tests to cover different conditions.
1. if holder === bene
2. approved bene address is there. 
3. if admin address === holder 
4. admin address == bene
5. if adminAddr !== bene and adminAddr !== holder